### ISSUE TITLE
Convert Current-Chapters.md to a CSV

### ISSUE THREAD

**Issue by ledell** [rladies/starter-kit#29]
This will make it easier to use this data in other applications (e.g. rladies.org website).  The best way to do it would be to write a quick script to parse the markdown file and turn it into a CSV.  A template for what the CSV should look like is here: https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv

**Comment by stephaniehicks** [comment]
I got started on this issue. I transformed the data into a data frame format with `Country` as the first column and everything else in the second column. Next, need to break this column down into `State`, `City`, etc. 

```
library(readr)
library(dplyr)
library(tidytext)
library(purrr)
url <- "https://raw.githubusercontent.com/rladies/starter-kit/master/Current-Chapters.md"
dat <- data_frame(readr::read_lines(url))
colnames(dat) <- "lines"
 
 dat %>% 
   unnest_tokens(word, lines, token="regex", pattern="\\s+#{2}+\\s") %>% 
   filter(!grepl("ctrl\\+f|\\saccount\\s", word)) %>% 
   map_df(stringr::str_replace, pattern = "!\\[\\]\\(.*?\\)\n\n", "") %>%
   tidyr::separate(col=word, into= c("Country", "Else"), sep="\n\n", extra="merge")
```

**Comment by chucheria** [comment]
@stephaniehicks let me help!

**Comment by chucheria** [comment]
@ledell what do you think about adding a column _status_? The badge thing of the .md is a good idea, this way we don't loose it.

**Comment by stephaniehicks** [comment]
Thanks @chucheria! I haven't had time to work on it and next week is really busy for me. If you could finish, that would be **awesome**

**Comment by ledell** [comment]
Looking good so far!  When the transition complete, I'd recommend that we remove the text from the .md file so we don't have two versions which can get out of sync.  If we want to leave it there for a while in case people have linked to the file elsewhere, we could just edit the file with an update that says: "Chapter data has been moved to Current-Chatpers.csv".  Or we can delete it... either way.

**Comment by chucheria** [comment]
@ledell I think the redirect is better, so people can find the new file (including us! I don't know where I've linked that file 😅)

The file is complete, @gdequeiroz and @lauracion can start using it from now on.

**Comment by gdequeiroz** [comment]
Great! Thank you all. I noticed that the md file was updated 2 days ago and the csv was updated 4 days ago so we will probably need to rerun the script. 

![image](https://user-images.githubusercontent.com/1934546/41735141-bbff14a2-753d-11e8-8ecb-f07d750e1445.png)

**Comment by ledell** [comment]
@chucheria Can you manually add the two changes that @gdequeiroz added in the past few days?  There are two commits that need to be added to the CSV.  Then we should remove all data from Current-Chapters.md and replace it with a redirect so no one edits it anymore.  Thanks!

- https://github.com/rladies/starter-kit/commit/62bc4310708206778fd35aaee34e0167089630cf
- https://github.com/rladies/starter-kit/commit/80a82678986c81fcc96d4b21253a5438b6269c33

**Comment by lauracion** [comment]
I will wait until a new "go for the .csv"  to onboard a few requests of new chapters that came in these past days. Thank you!

**Comment by chucheria** [comment]
I commited the changes plus the Sydney Github (which Danielle gave me for the social media page). I think both files have the same info now 😊

**Comment by lauracion** [comment]
Thank u, Bea! We should now start using the .csv, correct?

**Comment by chucheria** [comment]
The PR is not merged yet but yes!

**Comment by gdequeiroz** [comment]
Closed #34
